docs/notebooks/pre_executed/snana_example.ipynb

```diff
@@ -0,0 +1,298 @@
+{
```

Comment:
<p>Let's put some redshift distribution in? So we have realistic fluxes. We use uniform between 0.1 and 0.6 in another notebook, it is not realistic, but still better</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/lightcurvelynx/pull/573/discussion/#comment-2474972818'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>